admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -200,15 +200,15 @@ export default class OrderProductRenderer {
   toggleColumnLocation(forceDisplay = null) {
     let isColumnLocationDisplayed = false;
     if (forceDisplay === null) {
-      $(OrderViewPageMap.productsCellLocation).filter('td').each(function() {
+      $(OrderViewPageMap.productsCellLocationDisplayed).filter('td').each(function() {
         if ($(this).html() !== '') {
           isColumnLocationDisplayed = true;
           return false;
         }
       });
+      $(OrderViewPageMap.productsCellLocationDisplayed).toggleClass('d-none', !isColumnLocationDisplayed);
     } else {
-      isColumnLocationDisplayed = forceDisplay;
+      $(OrderViewPageMap.productsCellLocation).toggleClass('d-none', !forceDisplay);
```

Comment:
I'd say you're gonna have the same problem here than with the previous bug, you must only toggle `productsCellLocationDisplayed`